querydsl-codegen/src/main/java/com/querydsl/codegen/DefaultVariableNameFunction.java

```diff
@@ -0,0 +1,26 @@
+package com.querydsl.codegen;
+
+import com.google.common.base.Function;
+import com.mysema.codegen.StringUtils;
+import com.querydsl.core.util.JavaSyntaxUtils;
+
+/**
+ * Default variable name generation strategy which un-capitalizes the first letter of the variable name.
+ *
+ */
+public final class DefaultVariableNameFunction implements Function<EntityType, String> {
+
+    public static final DefaultVariableNameFunction INSTANCE = new DefaultVariableNameFunction();
+
+    private DefaultVariableNameFunction() {
```

Comment:
public empty constructor, otherwise we block the instantiation via clazz.newInstance()